docs/plugins/media-query-tracker.md

```diff
@@ -0,0 +1,232 @@
+    <td><code>'event'</code></td>
+  </tr>
+  <tr valign="top">
+    <td><a href="https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#eventCategory"><code>eventCategory</code></a></td>
+    <td><code>definition.name</code></a></td>
+  </tr>
+  <tr valign="top">
+    <td><a href="https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#eventAction"><code>eventAction</code></a></td>
+    <td><code>'change'</code></a></td>
+  </tr>
+  <tr valign="top">
+    <td><a href="https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#eventLabel"><code>eventLabel</code></a></td>
+    <td><code>options.changeTemplate()</code></td>
+  </tr>
+</table>
+
+**Note:** the reference to `definition` in the table above refers to the definition the changing media value is defined in. The reference to `options` refers to passed configuration [options](#options).
+
+## Methods
+
+The following table lists all methods for the `mediaQueryTracker` plugin:
+
+<table>
+  <tr valign="top">
+    <th align="left">Name</th>
+    <th align="left">Description</th>
+  </tr>
+  <tr valign="top">
+    <td><code>remove</code></a></td>
+    <td>Removes the <code>mediaQueryTracker</code> plugin from the specified tracker, removes all media query listeners, and restores all modified tasks to their original state prior to the plugin being required.</td>
+  </tr>
+</table>
+
+For details on how `analytics.js` plugin methods work and how to invoke them, see [calling plugin methods](https://developers.google.com/analytics/devguides/collection/analyticsjs/using-plugins#calling_plugin_methods) in the `analytics.js` documentation.
+
+## Example
+
+### Basic usage
+
+This example requires the `mediaQueryTracker` plugin and customizes it to track breakpoint, resolution, and orientation media query data:
+
+```js
+ga('require', 'mediaQueryTracker', {
+  definitions: [
+    {
+      name: 'Breakpoint',
+      dimensionIndex: 1,
+      items: [
+        {name: 'sm', media: 'all'},
+        {name: 'md', media: '(min-width: 30em)'},
+        {name: 'lg', media: '(min-width: 48em)'}
+      ]
+    },
+    {
+      name: 'Resolution',
+      dimensionIndex: 2,
+      items: [
+        {name: '1x',   media: 'all'},
+        {name: '1.5x', media: '(min-resolution: 144dpi)'},
+        {name: '2x',   media: '(min-resolution: 192dpi)'}
+      ]
+    },
+    {
+      name: 'Orientation',
+      dimensionIndex: 3,
+      items: [
+        {name: 'landscape', media: '(orientation: landscape)'},
+        {name: 'portrait',  media: '(orientation: portrait)'}
+      ]
+    }
+  ]
+});
+```
+
+### Customizing the change template and timeout
+
+This code updates the change template to only report the new media value in the event hit. It also eliminates the debounce timeout amount for change events, so hits are sent as soon and as often as changes occur (not recommended):
```

Comment:
instead of showing a non-recommended value, how about setting debounce to a different value?